tests/functional/src/main/resources/META-INF/mule-module.properties

```diff
@@ -19,6 +19,7 @@ artifact.export.classPackages=org.mule.functional,\
                               org.mule.functional.util.ftp,\
                               org.mule.functional.util.sftp,\
                               org.mule.tck.util,\
+                              org.mule.functional.junit4.runners,\
```

Comment:
Previous question was asking if everything in that package must be exported, as everything there will be API. If not every class there must be exported, it would be better to create subpackages to isolate them